source/fresh-installation.rst

```diff
@@ -30,16 +30,16 @@ The configuration in ``/opt/conda/.condarc`` designates this location as the sec
 place conda should look for environments, after ``~/conda_envs``.
 
 Environments for data collection and data anlysis should be installed into
-``/opt/conda_envs``. In the second operating cycle of 2017, the commands to do
+``/opt/conda_envs``. In the second operating cycle of 2018, the commands to do
 this were:
 
 .. code-block:: bash
 
-    sudo conda create -p /opt/conda_envs/collection-17Q2.0 -c nsls2-tag -y collection
-    sudo conda create -p /opt/conda_envs/analysis-17Q2.0 -c nsls2-tag -y analysis
+    sudo conda create -p /opt/conda_envs/collection-2018-2.1 -c nsls2-tag -y collection
+    sudo conda create -p /opt/conda_envs/analysis-2018-2.1 -c nsls2-tag -y analysis
```

Comment:
It's probably worth explaining the structure here. ("What does the `.1` mean?" a user will wonder.)